apps/src/code-studio/initializeCodeMirror.js

```diff
@@ -87,6 +88,17 @@ function initializeCodeMirror(target, mode, options = {}) {
             }),
             previewElement
           );
+        } else if (game === 'Aichat' || game === 'Music') {
+          ReactDOM.render(
+            React.createElement(InstructionsPanel, {
+              text: editor.getValue(),
+              theme: 'Light',
```

Comment:
Is Music Lab always dark mode? I could make this dark for the Music Lab case if preferable, although might be a little stark against the white of the level edit page.